lib/rack/utils.rb

```diff
@@ -222,7 +222,7 @@ def parse_cookies_header(header)
       end
     end
 
-    def add_cookie_to_header(header, key, value)
```

Comment:
The public methods that you would like to remove should be properly deprecated in Rack 3.0 before removal in Rack 3.1.  If you don't think that is feasible, please explain why for each case.

Reply:
I’ll do my best to add compatible shims.

Reply:
Okay, I've added compatibility shims.